pyproject.toml

```
@@ -1,5 +1,55 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "zyte-common-items"
+dynamic = ["version"]
+description = "Item definitions for Zyte API schema"
+readme = { file = "README.rst", content-type = "text/x-rst" }
+requires-python = ">=3.9"
+authors = [{ name = "Zyte Group Ltd", email = "info@zyte.com" }]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: Implementation :: CPython",
+]
+dependencies = [
+    "attrs>=22.2.0",
+    "clear-html>=0.4.0",
+    "itemadapter>=0.8.0",
+    "Jinja2>=2.7",
+    "price-parser>=0.3.4",
+    "web-poet>=0.14.0",
+    "zyte-parsers>=0.5.0",
+]
+
+[project.urls]
+source = "https://github.com/zytedata/zyte-common-items"
+issues = "https://github.com/zytedata/zyte-common-items/issues"
+documentation = "https://zyte-common-items.readthedocs.io/en/stable/"
+changelog = "https://github.com/zytedata/zyte-common-items/blob/main/CHANGELOG.rst"
+
+[tool.setuptools.dynamic]
+version = { file = "zyte_common_items/VERSION" }
+
+[tool.setup_tools.package-data]
+zyte_common_items = ["py.typed","VERSION"]
+
+[tool.setuptools.packages.find]
+exclude= ["tests", "docs*"]
```

```suggestion
include= ["zyte_common_items"]
```

When trying `python -m build` locally, I got a file from the venv (`venv/bin/jp.py`). This change fixes that, and this is probably a safer approach to avoid including other unrelated files in the future.

---

Makes sense